### PR TITLE
Getting started page

### PR DESCRIPTION
**Pull request by johnkuney**
adding getting started page
- added react-bootstrap to build for help with formatting as old site using bootstrap and makes for quick grid layout

### CODE REVIEW

File: v2/src/pages/start-here.js [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+                <img
+                  className={S.cardImg}
+                  src={GetStarted}
+                  alt="Getting Started Bitcoin Cash"
+                />
+              </Link>
+              <div className={S.cardContent}>
+                <p className={S.cardSubtitle}>
+                  <fbt desc="Getting Started Card 'What is' subtitle">
+                    What is peer to peer electronic cash?{" "}
```
Comment (deadalnix) [inline v2/src/pages/start-here.js:39]: remove trainling space

File: v2/src/pages/start-here.js [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+          <Col md={12}>
+            <h1 className={S.headeralign}>
+              <fbt desc="Getting Started Header">Getting Started</fbt>
+            </h1>
+          </Col>
+        </Row>
+        <Row>
+          <Col md={6}>
+            <div className={S.card}>
+              <Link to="getting-started.html">
```
Comment (deadalnix) [inline v2/src/pages/start-here.js:29]: /getting-started.html

File: v2/src/pages/start-here.js [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+                    to prevent fraud. This makes P2P cash a trustless and safe
+                    way to transact without the need of intermediaries.
+                  </fbt>
+                </p>
+              </div>
+
+              <div className={S.cardButtonContainer}>
+                <PrimaryButton
+                  noMarginLeft={true}
+                  buttonText={fbt("View More", "'View More' button")}
```
Comment (deadalnix) [inline v2/src/pages/start-here.js:66]: this description is not going to help translators.

File: v2/src/pages/start-here.js [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+              </Link>
+              <div className={S.cardContent}>
+                <p className={S.cardSubtitle}>
+                  <fbt desc="Getting Started Card 'Wallets' subtitle">
+                    Where do I store my Bitcoin Cash?
+                  </fbt>
+                </p>
+
+                <p className={S.cardTitle}>
+                  {" "}
```
Comment (deadalnix) [inline v2/src/pages/start-here.js:89]: Remove spaces

File: v2/src/pages/start-here.js [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+                </p>
+
+                <p className={S.cardTitle}>
+                  <fbt desc="Getting Started Card 'Buy' title">
+                    3. Buy or Earn Bitcoin Cash
+                  </fbt>
+                </p>
+
+                <p className={S.cardParagraph}>
+                  {" "}
```
Comment (deadalnix) [inline v2/src/pages/start-here.js:137]: dito

File: v2/src/pages/start-here.js [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+              </Link>
+              <div className={S.cardContent}>
+                <p className={S.cardSubtitle}>
+                  <fbt desc="Getting Started Card 'Spend' subtitle">
+                    What can I buy with Bitcoin Cash?
+                  </fbt>
+                </p>
+
+                <p className={S.cardTitle}>
+                  {" "}
```
Comment (deadalnix) [inline v2/src/pages/start-here.js:175]: remove

File: v2/src/pages/start-here.js [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+                  noMarginLeft={true}
+                  buttonText={fbt("View More", "'View More' button")}
+                  href={"/buy-bitcoin-cash.html"}
+                />
+              </div>
+            </div>
+          </Col>
+          <Col md={6}>
+            <div className={S.card}>
+              <Link to="spend-bitcoin-cash.html">
```
Comment (deadalnix) [inline v2/src/pages/start-here.js:160]: /spend-bitcoin-cash.html

File: v2/package.json [hunk 8]
```diff
@@ -29,6 +30,7 @@
     "node-sass": "^4.14.1",
     "prop-types": "^15.7.2",
     "react": "^16.12.0",
+    "react-bootstrap": "^1.3.0",
```
Comment (deadalnix) [inline v2/package.json:33]: Good call!

**Review by deadalnix — CHANGES_REQUESTED** [review]

File: v2/src/pages/start-here.js (hunk 1 above)
Comment (johnkuney) [reply v2/src/pages/start-here.js]: done

File: v2/src/pages/start-here.js (hunk 2 above)
Comment (johnkuney) [reply v2/src/pages/start-here.js]: Okay fixed

File: v2/src/pages/start-here.js (hunk 3 above)
Comment (johnkuney) [reply v2/src/pages/start-here.js]: fixed

File: v2/src/pages/start-here.js (hunk 4 above)
Comment (johnkuney) [reply v2/src/pages/start-here.js]: done

File: v2/src/pages/start-here.js (hunk 5 above)
Comment (johnkuney) [reply v2/src/pages/start-here.js]: done

File: v2/src/pages/start-here.js (hunk 6 above)
Comment (johnkuney) [reply v2/src/pages/start-here.js]: removed

File: v2/src/pages/start-here.js (hunk 7 above)
Comment (johnkuney) [reply v2/src/pages/start-here.js]: fixed

**Review by PlatinumWrist5055 — COMMENTED** [review]
This been reviewed

**Pull request merged**